### PR TITLE
[USING] add checkout branch

### PR DESCRIPTION
**Pull request by MaxHillebrand**
This adds a step by step guide on how to checkout a developer branch. 

I also fix some minor things in the existing work.

Ready for review and merge.

### CODE REVIEW

File: docs/using-wasabi/BuildSource.md [hunk 1]
```diff
@@ -13,13 +13,13 @@
...
 
-:::tip Optional
+:::tip Optional for privacy
 You can disable .NET's telemetry, which is sending some usage information to Microsoft, by typing on Linux and OSX: `export DOTNET_CLI_TELEMETRY_OPTOUT=1` or on Windows `set DOTNET_CLI_TELEMETRY_OPTOUT=1`.
 :::
 
 ## Get Wasabi
 
-Clone & Restore & Build
+Clone & restore & build
```
Comment (yahiheb) [inline docs/using-wasabi/BuildSource.md]: I am not sure but I think we can remove the word `restore` here.

**Comment by yahiheb** [comment]
I think we shouldn't encourage people to build from developers because a one can make critical changes in his repository that might end up in loss of fund.

File: docs/using-wasabi/BuildSource.md (hunk 1 above)
Comment (MaxHillebrand) [reply docs/using-wasabi/BuildSource.md]: I think with restore it means `cd WalletWasabi/WalletWasabi.Gui`?

**Comment by MaxHillebrand** [comment]
You are right @yahiheb, it should at least be a notice that the developer branches are not thoroughly reviewed yet and can contain critical bugs. But I still think we should explain to users how to do it, if they so choose, this is especially for testers and contributors.

File: docs/using-wasabi/BuildSource.md (hunk 1 above)
Comment (yahiheb) [reply docs/using-wasabi/BuildSource.md]: that command only changes the directory (cd) to `WalletWasabi/WalletWasabi.Gui`

File: docs/using-wasabi/BuildSource.md [hunk 2]
```diff
@@ -29,10 +29,30 @@ dotnet build
...
 ```
+
+## Checkout a developer branch
+
+Add the developer to your remote & fetch his work & checkout the branch & pull the latest commits
+
+```sh
+git remote add nopara73 https://github.com/nopara73/WalletWasabi
+git fetch nopara73
+git checkout dev-branch
```
Comment (yahiheb) [inline docs/using-wasabi/BuildSource.md]:
```suggestion
git checkout branchname
```

File: docs/using-wasabi/BuildSource.md [hunk 3]
```diff
@@ -29,10 +29,30 @@ dotnet build
...
+git pull
+```
+
+:::warning Experimental
+These developer branches are not yet thoroughly reviewed by other contributors, and there might be critical bugs in the software that can even cause loss of funds.
```
Comment (yahiheb) [inline docs/using-wasabi/BuildSource.md]:
```suggestion
These developer branches are not thoroughly reviewed by other contributors, and there might be critical bugs in the software that can even cause loss of funds.
```

File: docs/using-wasabi/BuildSource.md [hunk 4]
```diff
@@ -29,10 +29,30 @@ dotnet build
...
+So only compile these branches if you know what you are doing, and run them only on testnet.
```
Comment (yahiheb) [inline docs/using-wasabi/BuildSource.md]:
```suggestion
So only compile these branches if you know what you are doing, and run them at your own risk only on testnet.
```

**Pull request merged**